taier-worker/taier-worker-api/pom.xml

```diff
@@ -78,7 +78,7 @@
         <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
-            <version>1.1.7</version>
+            <version>1.2.0</version>
```

Comment:
this version has one or more vulnerabilities from dependencies
see:
`https://mvnrepository.com/artifact/ch.qos.logback/logback-classic/1.2.0`
[logback-classic:1.2.0 maven desc](https://mvnrepository.com/artifact/ch.qos.logback/logback-classic/1.2.0)

You can update it or solve it, if you choose to update it, you need to consider the compatibility between it and other dependent versions

View version differences between `1.1.7` and `1.2.0` see:
`https://github.com/qos-ch/logback/compare/v_1.1.7...v_1.2.0`
[version differences between `1.1.7` and `1.2.0`](https://github.com/qos-ch/logback/compare/v_1.1.7...v_1.2.0)